molecule/default/tests/test_default.py

```diff
@@ -29,7 +29,7 @@ def test_command(host, f):
     assert host.file(f).is_file
 
 
-@pytest.mark.parametrize("pkg", ["at", "jq"])
+@pytest.mark.parametrize("pkg", ["at", "jq", "python3-virtualenv"])
```

Comment:
If we're checking for `python3-virtualenv` then I think that is what we should be installing (instead of `virtualenv`).

Reply:
You are correct, sir!  Please let me know if commit d157ad8 is to your liking.

Reply:
Bonege!